### PR TITLE
pkp/pkp-lib#941 dashboard adjustments

### PR DESCRIPTION
**Pull request by NateWr**
[Companion PR](https://github.com/pkp/pkp-lib/pull/971)

### CODE REVIEW

File: locale/en_US/submission.xml [hunk 1]
```diff
@@ -44,6 +44,7 @@
 	<message key="submission.confirmSubmit">Are you sure you wish to submit this manuscript to the press?</message>
 	<message key="submission.metadata">Metadata</message>
 	<message key="submission.supportingAgencies">Supporting Agencies</message>
+	<message key="submission.grid.titleColumn">Author; Book Title</message.
```
Comment (beghelli) [inline locale/en_US/submission.xml]: @NateWr I think this should go into library locale files, with the "book" word removed, something like Author; Title. Usually we try to make the pkp library an encapsulated component, so if you use a locale key in pkp library, the locale key definition should also be in pkp library. That way it's much easier for us to maintain and use the code.

It it's a requirement to have the word that defines the kind of object (book, article, etc), then I would create a template method that defines the locale key for this to be implemented in the application level and use this template method in the library. Makes sense?

Comment (NateWr) [inline locale/en_US/submission.xml]: I agree but I couldn't find a good place to put the locale key in `pkp-lib`. Do you have any ideas? I struggle to really grasp the scope of the various files and I'm often not sure of the right place to put a locale key... I searched for grid-related strings but didn't find anything that looked appropriate.

Comment (beghelli) [inline locale/en_US/submission.xml]: at the end of the lib/pkp/locale/en_US/grid.xml file, you can create another section like  
and add that locale key, without the specific object name, of course.

**Comment by NateWr** [comment]
Thanks @beghelli! This PR is no longer needed as all the changes are in `pkp-lib`.

**Pull request closed without merging**